packages/router/src/router.tsx

```diff
@@ -68,34 +63,28 @@ function Route(props: RouteProps | RedirectRouteProps | NotFoundRouteProps) {
 
 const InternalRoute: React.VFC<InternalRouteProps> = ({
   path,
-  page,
   name,
   redirect,
   notfound,
 }) => {
   const routerState = useRouterState()
   const activePageContext = useActivePageContext()
 
+  /** The not-found route is handled by the router. */
   if (notfound) {
-    // The "notfound" route is handled by <NotFoundChecker>
     return null
   }
 
-  if (!path) {
-    throw new Error(`Route "${name}" needs to specify a path`)
```

Comment:
Unless I'm missing something I still think this would be a helpful error message to have. Especially for JS projects that might not get any compile time errors.

Reply:
This is now in the `flatChildArray` validation loop:
https://github.com/redwoodjs/redwood/blob/f26ac7ce2d97130f65f9732e45307d679ba64762/packages/router/src/router.tsx#L187-L189